.gitignore

```diff
@@ -275,3 +275,7 @@ WinSign.*
 ADV_INST_KEY.key
 WinSign.b64
 Installer/ZitiDesktopEdge.back.aip
+ziti-edge-tunnel/deps/
```

Comment:
you can just use /ziti-edge-tunnel/ instead of specifying all the folders/files/dirs. it's a bit easier